### ISSUE TITLE
Width calculation bug 

### ISSUE THREAD

**Issue by jmarceli** [xobotyi/react-scrollbars-custom#84]
Hi,
If you update LIVE DEMO to latest package version `4.0.5` you will see that text is not fully visible. Here is the live exmple https://codesandbox.io/s/rsc-live-example-h34of

This bug is most probably caused by the change from `padding: 0.05` to `display: table-cell` in this commit https://github.com/xobotyi/react-scrollbars-custom/commit/7c2d3b83429d2ed74d696a4e03ead5ae01fb172b

**Comment by xobotyi** [comment]
What for i've written an issue example?🤦‍♂️

Everything is good for me. What browser do you use?

**Comment by jmarceli** [comment]
HI, I'm using Google Chrome Version 75.0.3770.90 (Official Build) (64-bit) on Max OS X Mojave and it looks like this:

![image](https://user-images.githubusercontent.com/4281333/59626315-d68a5a00-913b-11e9-9386-cb683de53719.png)

**Comment by xobotyi** [comment]
Cause there is no possible way for me to test it on macos - check this one pls.
https://codesandbox.io/s/rsc-live-example-062ls

**Comment by jmarceli** [comment]
Hi, when I use `translateContentSizesToHolder={false}` it doesn't change anything besides `padding: 0.05px`, but it still looks not as I expect.

![image](https://user-images.githubusercontent.com/4281333/59637864-1e6aaa80-9157-11e9-983c-458afcddb2fc.png)

I will try to find a way to make it look ok and then I will add a solution to that issue (as soon as I will have some spare time).

**Comment by xobotyi** [comment]
Very weird...
Looks like the scrollbar width compensation is turned off =/

**Comment by jmarceli** [comment]
No, it's not the compensation. When I remove/hide scrollbar with CSS `display: none` the content still falls behind the right edge of the scrolling area.

**Comment by xobotyi** [comment]
Thus a have absolutely no clue what is going on...
Try to manually set the `left` style to 0 for the scroller element and ensure that native scrollbar is still there.

**Comment by jmarceli** [comment]
I will try to figure that out and return here with some solution

**Comment by xobotyi** [comment]
@jmarceli HI! Any results?

**Comment by xobotyi** [comment]
Looks like common linux is affected too.

**Comment by jmarceli** [comment]
It seems to be still an issue. I've checked demo page https://xobotyi.github.io/react-scrollbars-custom/ and it looks like this:

<img width="576" alt="macosx-chrome" src="https://user-images.githubusercontent.com/4281333/66069120-04f73e00-e54f-11e9-9e22-bed8ce1e6505.png">

This is taken from Chrome Version 76.0.3809.132 on Mac OS X 10.14, but to be honest I'm no longer tracking this issue as I've changed scrollbars library (sorry).

**Comment by xobotyi** [comment]
@jmarceli to which one if not secret? It hasnt that issue?

**Comment by jmarceli** [comment]
https://github.com/KingSora/OverlayScrollbars with react adapter https://github.com/KingSora/OverlayScrollbars/tree/master/packages/overlayscrollbars-react while it is not perfectly documented I manage to make it work pretty well for my project.

**Comment by KingSora** [comment]
Good day!

I want to contribute to this bug. This bug isn't platform specific but happens if the browser has already overlaid scrollbars (e.g. the native scrollbar-width is 0). You can simulate this behavior on any system with chrome. Go to `chrome://flags/` and enable the option 'Overlay Scrollbars':
![gh](https://user-images.githubusercontent.com/12936317/72909964-1e0f3f00-3d38-11ea-8e09-576038b46b77.png)

To provide a clue on why this is happening:
Your compensation strategy with `margin-right: -20px` and `padding-right: 20px` isn't working if the horizontal axis is overflowing.

@jmarceli I'm the author of `OverlayScrollbars` and I'm glad the plugin is working for you. May I ask you how I could improve the documentation? In my eyes I've done a pretty good job with it, but maybe there are certain thing which are too hidden for new users like you was. Thanks for your input!

**Comment by vdjurdjevic** [comment]
@KingSora I think I have a similar issue (using your lib with React). I have a grid layout with a column template auto 1fr auto. My scrollable element is in the middle. Inside the scrollable element, I have expansion panels each having number of cards. When all panels are collapsed, the element width is good. Here is a picture: 
![image](https://user-images.githubusercontent.com/17208031/80163083-4fb4d200-85d5-11ea-9719-2c7ae79460d4.png)

Now, if I expand the last panel which fits in the viewport (not causing overflow), width increases for some reason. Here is a picture (notice how third column is clipped): 
![image](https://user-images.githubusercontent.com/17208031/80163174-97d3f480-85d5-11ea-95d2-dbdfef43c623.png)

And if I expand two panels, there is enough content to cause overflow, everything is ok. Here is a picture: 

![image](https://user-images.githubusercontent.com/17208031/80163243-dcf82680-85d5-11ea-8fa4-be6819761e0f.png)

Any ideas why is this happening?

**Comment by KingSora** [comment]
@vdjurdjevic please open a issue in my repo for this case, I don't really wanna spam someone others issues with unrelated stuff to his project.
